TestSTMConflict fails with invalid argument to Intn

```
=== RUN   TestSTMConflict
panic: invalid argument to Intn [recovered]
	panic: invalid argument to Intn

goroutine 72815 [running]:
github.com/coreos/etcd/clientv3/concurrency.runSTM.func1.1(0xc821bcb0e0)
	/home/runner/workspace/src/github.com/coreos/etcd/gopath/src/github.com/coreos/etcd/clientv3/concurrency/stm.go:72 +0xab
math/rand.(*Rand).Intn(0xc8200146a0, 0x0, 0x0)
	/usr/local/go/src/math/rand/rand.go:98 +0x79
math/rand.Intn(0x0, 0x3)
	/usr/local/go/src/math/rand/rand.go:198 +0x2d
github.com/coreos/etcd/integration.TestSTMConflict.func1(0x7f4f44f75778, 0xc8204eb400, 0x0, 0x0)
	/home/runner/workspace/src/github.com/coreos/etcd/gopath/src/github.com/coreos/etcd/integration/v3_stm_test.go:56 +0x221
github.com/coreos/etcd/clientv3/concurrency.runSTM.func1(0xc821bcb0e0, 0x7f4f44f75778, 0xc8204eb400, 0xc821476660)
	/home/runner/workspace/src/github.com/coreos/etcd/gopath/src/github.com/coreos/etcd/clientv3/concurrency/stm.go:80 +0x84
created by github.com/coreos/etcd/clientv3/concurrency.runSTM
	/home/runner/workspace/src/github.com/coreos/etcd/gopath/src/github.com/coreos/etcd/clientv3/concurrency/stm.go:88 +0x84

...
```


https://semaphoreci.com/coreos/etcd/branches/pull-request-4871/builds/3

--- a/integration/v3_stm_test.go
+++ b/integration/v3_stm_test.go
@@ -53,6 +53,10 @@ func TestSTMConflict(t *testing.T) {
 			dst := stm.Get(dstKey)
 			srcV, _ := strconv.ParseInt(src, 10, 64)
 			dstV, _ := strconv.ParseInt(dst, 10, 64)
+			if srcV == 0 {
+				// can't rand.Intn on 0, so skip this transaction
+				return nil
+			}
 			xfer := int64(rand.Intn(int(srcV)) / 2)
 			stm.Put(srcKey, fmt.Sprintf("%d", srcV-xfer))
 			stm.Put(dstKey, fmt.Sprintf("%d", dstV+xfer))

